pkgs/os-specific/linux/alsa-firmware/cross.patch

```diff
@@ -0,0 +1,347 @@
+--- a/hdsploader/Makefile.am	2015-02-26 20:36:03.000000000 +0800
```

Comment:
Where did you get this patch from?

Reply:
I composed this patch myself. :sweat_smile: